src/api/errors/levels/FatalError.ts

```diff
@@ -0,0 +1,20 @@
+import BaseError from './BaseError'
+
+class FatalError extends BaseError {
+  cause?: Error
+
+  constructor (title: string, message: string, cause?: Error) {
+    super(500, title, message, 'fatal')
+    this.type = 'FatalError',
+    this.cause = cause
+  }
+
+  toJSON () {
```

Comment:
Parentheses must be attached to the method name.
```suggestion
  toJSON() {
```